vehicles/src/common/enum/credit-accounts.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum CreditAccountUserType {
+  ACCOUNT_OWNER = 'OWNER',
```

Comment:
Shall we rename it to ACCOUNT_HOLDER  = 'HOLDER' so that we remain consistent with Business requirement? Either way works